### PR TITLE
Adds news item feed for users

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-1225

### What?

Surface news items for all services for the update target under /feed/news_items

I have added/removed/altered:

- [x] Added atom feed for news items

### Why?

I am doing this because:

- This is a feature request

### CODE REVIEW

File: app/views/feed/news_items/index.atom.builder [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      entry.content      news_item.content
+      entry.start_date   news_item.start_date.try(:to_s, :rfc822) || ''
+      entry.end_date     news_item.end_date.try(:to_s, :rfc822) || ''
+      entry.uk           news_item.show_on_uk
+      entry.xi           news_item.show_on_xi
+      entry.home_page    home_url
+      entry.updates_page news_items_url
+    end
+  end
+end
```
Comment (alexdesi) [inline app/views/feed/news_items/index.atom.builder:19]: is it ok this file not to have .RB extension?

File: app/models/news_item.rb [hunk 2]
```diff
@@ -38,6 +38,13 @@ def updates_page
       )
     end
 
+    def for_feed
+      collection(
+        collection_path,
+        target: 'updates',
+      )
+    end
```
Comment (alexdesi) [inline app/models/news_item.rb:46]: it might be worth adding a test for `for_feed`

**Review by alexdesi — APPROVED** [review]
✅ LGTM

File: app/models/news_item.rb (hunk 2 above)
Comment (willfish) [reply app/models/news_item.rb:46]: Good idea

File: app/views/feed/news_items/index.atom.builder (hunk 1 above)
Comment (willfish) [reply app/views/feed/news_items/index.atom.builder:19]: Yeah, this is right :)

**Pull request merged**